src/classes/RD2_EntryFormController.cls

```diff
@@ -86,7 +86,8 @@ public with sharing class RD2_EntryFormController {
             'InstallmentPeriodPermissions' => getFieldPermissionsMap('npe03__Installment_Period__c'),
             'InstallmentFrequencyPermissions' => getFieldPermissionsMap(UTIL_Namespace.StrTokenNSPrefix('InstallmentFrequency__c')),
             'customFieldSets' => getCustomFieldSectionFields(),
-            'isElevateCustomer' => RD2_ElevateIntegrationService.isIntegrationEnabled()
+            'isElevateCustomer' => RD2_ElevateIntegrationService.isIntegrationEnabled(),
+            'isPilotEnabled' => UTIL_FeatureEnablement.isPilotEnabled()
```

Comment:
I recommend matching the pattern implemented by @andrewyu-salesforce here - https://github.com/SalesforceFoundation/NPSP/blob/master/src/classes/RD2_ElevateInformation_CTRL.cls#L65

Reply:
I like it! Will update this afternoon.